bin/trond

```diff
@@ -168,10 +169,10 @@ def setup_environment(args):
 
 def main():
     args = parse_cli()
-
+    tron_start_time = time.time()
```

Comment:
can we re-use the `boot_time` naming that already exists? i think in mcp.py we have a variable that we call `boot_time` for when the mcp object is created (might actually be nice to unify that with this, actually)

Reply:
yea, I was using it at first but then added tron_start_time to start timing as soon as we enter main instead of waiting till we get into "MasterControlProgram" class